bluemira/codes/_freecadapi.py

```diff
@@ -1416,37 +1462,165 @@ def exporter(self) -> ExporterProtocol:
         FreeCADError
             Unable to save file type
         """
-        try:
-            export_func = __import__(self.module).export
-        except AttributeError:
-            modlist = self.module.split(".")
-            if len(modlist) > 1:
-                return getattr(__import__(modlist[0]), modlist[1]).export
-            raise FreeCADError(
-                f"Unable to save to {self.value} please try through the main FreeCAD GUI"
-            ) from None
-        except TypeError:
+        if self.export_module is None:
             # Assume CADFileType.FREECAD
             def FreeCADwriter(objs, filename, **kwargs):  # noqa: ARG001
                 doc = objs[0].Document
                 doc.saveAs(filename)
 
             return FreeCADwriter
-        else:
-            if self in self.manual_mesh_formats():
-                return meshed_exporter(self, export_func)
-            return export_func
+        modlist = self.export_module.split(".")
+        try:
+            export_func = (
+                getattr(
+                    __import__(".".join(modlist[:-1]), fromlist=modlist[1:]),
+                    modlist[-1],
+                ).export
+                if len(modlist) > 1
+                else __import__(self.export_module).export
+            )
+        except AttributeError:
+            raise FreeCADError(
+                f"Unable to save to {self.value} "
+                "please try through the main FreeCAD GUI"
+            ) from None
+        if self in self.manual_mesh_formats():
+            return meshed_exporter(self, export_func)
+        if self is self.WEBGL:
+            return webgl_export(export_func)
+        return export_func
+
+    @DynamicClassAttribute
+    def importer(self) -> ImporterProtocol:
+        """Get importer module for each filetype
+
+        Raises
+        ------
+        FreeCADError
+            Unable to import file type
+        """
+        if self.import_module is None:
```

Comment:
How would you prefer?
Because the enum is set up as `(extension, export module, import module)` i'd need to pull in another variable from somewhere to be consistent